packages/video-sdk/src/video.ts

```diff
@@ -0,0 +1,113 @@
+export enum VIDEO_ACTIONS {
```

Comment:
@matheusmorett2 not sure you need the `enum` here, maybe just 

```tsx
type VideoActions = 'mute' | 'unmute' | 'play' | 'pause' // etc.
type Message = { action: VideoAction } | { action: 'seek', value: number }

// utility method with type
private sendMessage(message: Message) {
  this.iframe?.contentWindow?.postMessage(message, '*')
}

// then the methods are DRY too:

public seekTo(time: number) {
  sendMessage({ action: 'seek', value:time})
}

public mute() {
  sendMessage({ action: 'mute' })
}

Reply:
Good catch!